app/models/request_log_record.rb

```diff
@@ -0,0 +1,6 @@
+class RequestLogRecord < ApplicationRecord
```

Comment:
Is the word "Record" a recognized word to mean abstract or base class? Maybe consider "RequestLogBase"

Reply:
It's to do with 'ApplicationRecord', but agree RequestLogBase sounds better